Source/Scene/ShadowMap.js

```diff
@@ -185,8 +185,9 @@ define([
         this._needsUpdate = true;
 
         // In IE11 polygon offset is not functional.
+        // TODO : Also disabled for Chrome temporarily. Re-enable once https://groups.google.com/forum/#!topic/webgl-dev-list/E1dAG65QBhg is resolved.
```

Comment:
Write up a Cesium bug instead of linking to a forum post. (the Cesium bug can include the above link, of course).

If we ever need to do this check in more than one place, we should move the code to `FeatureDetection` but it's fine here for now.